### ISSUE TITLE
Unable to lock screen

### ISSUE THREAD

**Issue by Donkere-vader** [aunetx/blur-my-shell#3]
When this extension is turned on I can't lock the screen. The title bar and dock still work but i can't interact with running programs anymore, however they do still work

OS: Pop os 20.04 (based on Ubuntu 20.04)

**Comment by aunetx** [comment]
Thanks for the issue! I will have a look when I have my computer back :)

**Comment by slottwo** [comment]
I've been having a real problem, and my OS is also Pop_OS! 20.04 LTS

**Comment by aunetx** [comment]
Thanks for reporting, I experience this issue to... I will try to investigate to find the cause quickly

**Comment by slottwo** [comment]
If you need more information I can try to help

**Comment by Artofaj** [comment]
I can confirm this issue. This also prevents restarting the shell.

**Comment by DenishKakadiya** [comment]
Same issue was occurring in my machine. This extension is causing problem with lock screen.

**Comment by aunetx** [comment]
I'm sorry you all, I really don't know where it is coming from : only one month ago, everything worked...

I don't have any time to fix this now sadly, but if you want to test this with differents gnome versions, make sure to tell me about your results!

And i will try to fix this asap, I promise

**Comment by aunetx** [comment]
I created a fix, it should work fine now! Tell me if you experience any issue, the extension should be updated on extensions.gnome.org in some days.

**Comment by Donkere-vader** [comment]
It is indeed fixed now! GJ

**Comment by aunetx** [comment]
Cool :))

**Comment by slottwo** [comment]
For me the problem persists. But from what I understood by exploring other extensions that involved transparency and blur, it seems to me to be a problem with the window's shadows on gnome.

**Comment by aunetx** [comment]
Thanks for reporting, I will test it again... Could you transmit me your gnome-shell logs just when you lock your computer?

**Comment by slottwo** [comment]
Do you want the log while the pc is locked or after?
Is the log you want the one returned by the ```journalctl _UID``` command?

**Comment by slottwo** [comment]
Before: Logs begin at Fri 2020-10-09 13:43:06 -03, end at Fri 2020-10-09 13:52:23 -03
After: Logs begin at Fri 2020-10-09 13:43:06 -03, end at Fri 2020-10-09 14:04:04 -03
[Logs.zip](https://github.com/aunetx/blur-my-shell/files/5356457/Logs.zip)
Reading the logs, I think it may be a problem with the gnome shadows. But I don't know how to disable them.

**Comment by slottwo** [comment]
I recorded the error. Look at the top bar and see how the window "draws" on it. I know, the quality is not so good, I'm sorry.
[Visual bug video](https://drive.google.com/file/d/1NQRAyvLOlhMouZiDPSwpL8yJ4a0AoI8x/view?usp=sharing)

**Comment by slottwo** [comment]
Now that I reread the comment that started this issue, I realized that I made a huge misinterpretation.
I had understood that Donkere-vader's problem was that the taskbar and dock did not work well due to the interaction of the windows with them. I'm so sorry. 
It would be good for me to redo these comments in a new issue, right?

**Comment by aunetx** [comment]
Ok not problem! Thanks for reporting anyway, I'm working on improving this extension somehow (so maybe some bug will be fixed soon?)
